lib/src/api.ts

```diff
@@ -1,8 +1,11 @@
 import axios from "axios";
 import { Account } from "@ledgerhq/wallet-api-client";
 import BigNumber from "bignumber.js";
+import { decodeSellPayload } from "@ledgerhq/hw-app-exchange";
+import { BEData } from "./sdk";
 
 const SWAP_BACKEND_URL = "https://swap.ledger.com/v5/swap";
+const SELL_BACKEND_URL = "https://buy.api.aws.prd.ldg-tech.com/sell/v1/sell";
```

Comment:
If we have multiple endpoints...It won't be better add the root as a base url? 
```suggestion
const SELL_BACKEND_URL = "https://buy.api.aws.prd.ldg-tech.com/sell/v1";
```

Reply:
Exactly what i was working on refactoring! Great suggestion, already added!